wrapper-generator/src/test/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/VersionSuggestionsTest.kt

```diff
@@ -0,0 +1,35 @@
+package it.krzeminski.githubactions.wrappergenerator.generation
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import it.krzeminski.githubactions.wrappergenerator.domain.ActionCoords
+import it.krzeminski.githubactions.wrappergenerator.domain.WrapperRequest
+import it.krzeminski.githubactions.wrappergenerator.suggestDeprecations
+
+class VersionSuggestionsTest : FunSpec({
```

Comment:
Once the new logic reaches the new file:
```suggestion
class DeprecationSuggestionsTest : FunSpec({
```